manifests/database.pp

```diff
@@ -22,7 +22,8 @@
     }
 
     foreman::rake { 'db:migrate':
-      unless => '/usr/sbin/foreman-rake db:abort_if_pending_migrations',
+      timeout => 0,
```

Comment:
I am not 100% sure 0 is a good value here, but couldn't think of anything else.

Reply:
You could make it a parameter on the (private) class so you at least have a way to change it as a local workaround.

Reply:
You mean a param of the `foreman::database` class?

Reply:
Exactly.

Reply:
So, like this?